src/components/Addbox.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from "react";
+import Box from "./Box";
+class Addbox extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      lightArray: [],
+    };
+  }
+
```

Comment:
I think you can technically write these functions without the if statement, since there is only one condition. But I did the same thing as you.